### ISSUE TITLE
Show CAB Power Ratings in a few places

### ISSUE THREAD

**Issue by ineffyble** [stt-datacore/website#210]

**Comment by ylluminate** [comment]
What is the value of CAB? It seems quite variable and doesn't line up with crew I thought were more valuable, etc.

**Comment by ineffyble** [comment]
You can find info on CAB STT Power Ratings at https://sttpowerratings.com

**Comment by ylluminate** [comment]
Hmm: https://sttpowerratings.com/about-the-system/

Doesn't seem to quite line up with what I was used to seeing, but essentially the higher they are in the rank there (or now https://datacore.app/ > Crew Stats > sort by CAB) the more overall valuable they are?

**Comment by ineffyble** [comment]
That's the premise. Obviously there's no single right or wrong way to judge value, which is a matter of opinion and contextual to the roster. 

I'm hoping we can continue to add more data and rankings to DataCore to help people judge value by what's important to them.

**Comment by ylluminate** [comment]
Interesting, thanks. It does at least give some new indications for some value of crew beyond what was hitherto perceived. I have some new leveling targets now I suppose that I did not previously consider important or a priority.

**Comment by nickwolf** [comment]
I would definitely love the CAB ranking as a column on the crew retrieval page, particularly since the tier system is defunct.

**Comment by ylluminate** [comment]
Yeah, this is becoming more and more important to see on each area that includes any crew info at all.
